account_invoice_template/__init__.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 ##############################################################################
-#
+#    Copyright (C) 2017 Aurium Technologies (<http://www.auriumtechnologies.com>)
```

Comment:
You shouldn't add your copyright in files that you don't touch (and more if they are `__init__.py` files.